src/components/pre-playback-play-overlay/_pre-playback-play-overlay.scss

```diff
@@ -36,3 +36,22 @@
     display: none;
   }
 }
+
+.player{
+  &.size-ty {
+    .pre-playback-play-button {
+      width: 75px;
+      height: 75px;
+      padding: 12px;
+      margin: -35px 0 0 -42px;
```

Comment:
should be half of the width/height 
`margin: -37.5px 0 0 -37.5px;`
But remove the margin at all and add to the default:
`transform: translate(-50%, -50%);`